wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -943,13 +943,8 @@ impl PhysicalDeviceProperties {
         extensions.push(khr::swapchain::NAME);
 
         if self.device_api_version < vk::API_VERSION_1_1 {
-            // Require either `VK_KHR_maintenance1` or `VK_AMD_negative_viewport_height`
-            if self.supports_extension(khr::maintenance1::NAME) {
-                extensions.push(khr::maintenance1::NAME);
-            } else {
-                // `VK_AMD_negative_viewport_height` is obsoleted by `VK_KHR_maintenance1` and must not be enabled alongside it
-                extensions.push(amd::negative_viewport_height::NAME);
-            }
+            // Require `VK_KHR_maintenance1`
+            extensions.push(khr::maintenance1::NAME);
```

Comment:
question: Should this require a `CHANGELOG` entry? I imagine we fell back to `VK_AMD_negative_viewport_height` for a reason.

Reply:
issue: Ah, actually, this _whole thing_ is missing a `CHANGELOG` entry and is a pretty big deal, too. 😅

Trusting that this will be handled, despite having approved already.

Reply:
It's mostly old drivers that don't support `VK_KHR_maintenance1` but support `VK_AMD_negative_viewport_height` https://github.com/gpuweb/gpuweb/issues/4251#issuecomment-1656125294 - but I can note it in the changelog entry.